osu.Game/Overlays/MusicController.cs

```diff
@@ -456,6 +460,12 @@ public double CurrentTime
                 set { CurrentNumber.Value = value; }
             }
 
+            public bool Enabled

```

Comment:
A property with a set and get being simple like this is not needed. You can save the `enabled` variable and just use `Enabled` as a normal public bool